### DISCUSSION TITLE
Text Customization

### DISCUSSION THREAD

**Discussion by LucidAPs** [GriefPrevention/GriefPrevention#1168] [Ideas]
Ability to add a second line in messages.yml with /n 
Ability to customize the appearance of admin notifications, especially for signs. 
Also the /gpreload command doesn't refresh the messages.yml

**Comment by Jikoo** [comment]
You can already add newlines. Google "yaml newline in string" for information on how to use yaml syntax.
Pretty sure there's already a more generic "make all strings customizable" request but I can't find it.
Separate issues/discussions make it way easier to track this stuff - I guarantee you I will not remember that "customize more messages" includes "fix /gpreload not reloading messages" and I would definitely consider that a bug.

**Reply by RoboMWM** [reply]
/gpreload doesn't do a lot of things. Of course my workflow for servers is to always restart (yes, even in dev - though I do cheat sometimes with plugman) so idk how much I'd prioritize this, if any at all.

**Reply by Jikoo** [reply]
Yeah, I'm also of the mind that the message for it shouldn't include suggesting /reload, but it's low down the list of things I'd like to tackle
